docs/scripts_md/Notify.md

```diff
@@ -0,0 +1,112 @@
+# NAME
+
+NeuroDB::Notify -- Provides an interface to the email notification subsystem
+of LORIS
+
+# SYNOPSIS
+
+    use NeuroDB::Notify;
+
+    my $notifier = NeuroDB::Notify->new(\$dbh);
+
+    my $message           = "Some kind of message from tarchive validation";
+    my $upload_id         = 123456;
+    my $notify_notsummary = 'N';
+    $notifier->spool('tarchive validation', $message,   0,
+                             'tarchiveLoader',      $upload_id, 'Y',
+                             $notify_notsummary
+                            );
+
+# DESCRIPTION
+
+This class defines an interface into the email notification subsystem of
+LORIS - particularly with regards to spooling new messages.
+
+## Methods
+
+### new($dbh) (constructor)
+
+Creates a new instance of this class. The parameter `\$dbh` is a
+reference to a DBI database handle, used to set the object's database
+handle, so that all the DB-driven methods will work.
+
+INPUT: DBI database handle
+
+RETURNS: new instance of this class.
+
+### spool($type, $message, $centerID, $origin, $processID, ...)
+
+Spools a new notification message, `$message`, into the spool for notification
```

Comment:
into the spool 
is meant to mean 
into the `notification_spool` table?

Reply:
I guess. Zia wrote this.